pages/settings/admin/tasks.vue

```diff
@@ -0,0 +1,248 @@
+<template>
```

Comment:
Seperate PR please 😉

Reply:
Fine for me but it might look akward UI-wise as it depends on the changes made to ``SettingsPage`` component.

Reply:
That's fine